### PR TITLE
Remove the remnants of envify and properly load env vars in webpack.

### PR DESCRIPTION
**Pull request by kentquirk**
@jebeck -- I believe this resolves the problem we talked about. At least it seems to on my machine.

### CODE REVIEW

**Comment by jebeck** [comment]
Will have a look before your morning tomorrow @kentquirk 

On my way out the door right now.

**Comment by kentquirk** [comment]
I have deployed to devel and staging from this branch to make sure it's working, and it does in fact seem to work every time. I also inspected the configs on both branches of staging to make sure they had the same name and weren't filled up with 45K of ugly fat.

I'm happy with this one. Email forthcoming.

File: config.app.js [hunk 1]
```diff
@@ -35,12 +35,14 @@ function integerFromText(value, defaultValue) {
...
   VERSION: pkg.version,
-  MOCK: booleanFromText(process.env.MOCK, false),
-  MOCK_PARAMS: process.env.MOCK_PARAMS || '',
-  UPLOAD_API: process.env.UPLOAD_API || 'https://chrome.google.com/webstore/detail/tidepool-uploader/cabklgajffclbljkhmjphejemhpbghfb',
-  API_HOST: process.env.API_HOST || 'https://devel-api.tidepool.io',
-  SHOW_ACCEPT_TERMS: booleanFromText(process.env.SHOW_ACCEPT_TERMS, true),
-  PASSWORD_MIN_LENGTH: integerFromText(process.env.PASSWORD_MIN_LENGTH, 8)
+  MOCK: booleanFromText(__MOCK__, false),
+  MOCK_PARAMS: __MOCK_PARAMS__ || '',
+  UPLOAD_API: __UPLOAD_API__ || 'https://chrome.google.com/webstore/detail/tidepool-uploader/cabklgajffclbljkhmjphejemhpbghfb',
```
Comment (jebeck) [inline config.app.js]: I just noticed this...now that we have a devel Chrome app, shouldn't the default be the devel link, parallel to how devel-api is the default `API_HOST`? Given the context menu switching in the uploader, it doesn't really matter, just wondering.

**Comment by jebeck** [comment]
One question, but nothing to prevent a merge. LGTM.

**Comment by kentquirk** [comment]
Actually, we shouldn't have a reference to the Chrome Store at all. I've changed it to the uploader link.

**Pull request merged**